tests/distributed/test_partition.py

```diff
@@ -20,48 +24,67 @@ def _verify_partition_data_types(part_g):
         if k in part_g.edata:
             assert part_g.edata[k].dtype == dtype
 
+
+def _verify_partition_formats(part_g, formats):
+    # verify saved graph formats
+    if formats is None:
+        assert "coo" in part_g.formats()["created"]
+    else:
+        for format in formats:
+            assert format in part_g.formats()["created"]
+
+
 def _get_inner_node_mask(graph, ntype_id):
     if dgl.NTYPE in graph.ndata:
-        dtype = F.dtype(graph.ndata['inner_node'])
-        return graph.ndata['inner_node'] * F.astype(graph.ndata[dgl.NTYPE] == ntype_id, dtype) == 1
+        dtype = F.dtype(graph.ndata["inner_node"])
+        return (
+            graph.ndata["inner_node"]
+            * F.astype(graph.ndata[dgl.NTYPE] == ntype_id, dtype)
```

Comment:
Create a local variable for F.astype(graph.ndata[dgl.NTYPE] == ntype_id, dtype) to make the block more readable?

Reply:
Use same ones from `dgl.distributed.partition` instead